docs/source/doc-proj-alpha.rst

```diff
@@ -0,0 +1,37 @@
+======================================
+Project Documentation (alphabetically)
+======================================
+
+**Documentation on usage, configuration and development**
+
+
+* `configurable-http-proxy <https://github.com/jupyter/configurable-http-proxy>`_
+* `docker-stacks <https://github.com/jupyter/docker-stacks>`_
+* `dockerspawner <https://github.com/jupyter/dockerspawner>`_
+* `ipykernel <https://ipython.readthedocs.org/en/stable/>`_
+* `ipyparallel <https://ipyparallel.readthedocs.org/en/latest/>`_
+* `ipywidgets <https://ipywidgets.readthedocs.org/en/latest/>`_
+* `Jupyter Notebook <http://jupyter-notebook.readthedocs.org/en/latest/>`_
+* `jupyter-drive <https://github.com/jupyter/jupyter-drive>`_
+* `jupyter-js-notebook <https://github.com/jupyter/jupyter-js-notebook>`_
+* `jupyter-js-phosphide <https://github.com/jupyter/jupyter-js-phosphide>`_
+* `jupyter-js-plugins <https://github.com/jupyter/jupyter-js-plugins>`_
+* `jupyter-js-services <http://jupyter.org/jupyter-js-services/>`_
+* `jupyter-js-ui <http://jupyter.org/jupyter-js-ui/>`_
+* `jupyter-js-utils <http://jupyter.org/jupyter-js-utils/>`_
+* `jupyter-sphinx-theme <https://github.com/jupyter/jupyter-sphinx-theme>`_
+* `jupyter_client <http://jupyter-client.readthedocs.org/en/latest/>`_
+* `jupyter_console <http://jupyter-console.readthedocs.org/en/latest/>`_
+* `jupyter_core <http://jupyter-core.readthedocs.org/en/latest/>`_
+* `JupyterHub <http://jupyterhub.readthedocs.org/en/latest/>`_
+* `kernel_gateway <http://jupyter-kernel-gateway.readthedocs.org/en/latest/>`_
+* `ldapauthenticator <https://github.com/jupyter/ldapauthenticator>`_
+* `nbconvert <http://nbconvert.readthedocs.org/en/latest/>`_
+* `nbformat <http://nbformat.readthedocs.org/en/latest/>`_
+* `nbgrader <http://nbgrader.readthedocs.org/en/latest/>`_
+* `nbviewer <https://github.com/jupyter/nbviewer>`_
+* `oauthenticator <https://github.com/jupyter/oauthenticator>`_
+* `qtconsole <https://qtconsole.readthedocs.org/en/stable/>`_
+* `sudospawner <https://github.com/jupyter/sudospawner>`_
+* `tmpnb <https://github.com/jupyter/tmpnb>`_
+* `traitlets <http://traitlets.readthedocs.org/en/stable/>`_
```

Comment:
Having two indices like this seems like an invitation for them to get out of sync. I'd lose the alphabetic one - if you know what you're looking for you can search, and if you don't, the categorised one is much more helpful./